Python/marshal.c

```diff
@@ -1789,6 +1837,8 @@ marshal.dumps
     version: int(c_default="Py_MARSHAL_VERSION") = version
         Indicates the data format that dumps should use.
     /
+    *
+    stable: bool = True
```

Comment:
It would probably make sense to default to `False` for now, so users would effectively opt-in to the performance penalty.  (The same goes for `marshal.dump()`.)